bookshelf/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: custom
+vm: true
+
+manual_scaling:
```

Comment:
I also see the downside of having 2 instances. When you have trouble in your app, how do you determine which server to ssh into? I still think we should start from 1 instance and change the scaling settings if necessary.